consensus/consensus-types/src/utils.rs

```diff
@@ -0,0 +1,96 @@
+#[derive(Debug, Clone, Copy)]
+pub struct PayloadTxnsSize {
+    pub count: u64,
+    pub bytes: u64,
+}
+
+impl PayloadTxnsSize {
+    pub fn new(count: u64, bytes: u64) -> Self {
+        assert!(count <= bytes);
```

Comment:
I'm not sure we want this assert here. I think some of the backpressure, etc. will focus on reducing only count or bytes, without necessarily accounting for a txn should at least be one byte.

Reply:
Based on offline discussion, using PayloadTxnsSize in PayloadGenerator completely. The `consensus_config` still has two separate fields for count and bytes because that seems like a unnecessary and harder migration. Once PayloadGenerator converts the config fields to PayloadTxnsSize and uses it throughout, so any issues should surface at node startup, and not after that. I made sure all further operations are safe.